### PR TITLE
Local storage

### PR DESCRIPTION
**Pull request by gisachris**
### IN This PR I have done the following 👨‍💻 :
⚫  Adjust the contact form 📞  with javascript 🟨 to have local storage of user input until submited 🚀 .
⚫  Add some styling to better fit figma design 🎨  and overall website functionality 🧑‍🔧 

-implement the following actions 💥 :
**-When the user changes the content of any input field, the data is saved to the local storage..✅**
**-When the user loads the page, if there is any data in the local storage the input fields are pre-filled with this data. ✅**

I am looking forward to your comments on the work I have done so far 🌟 . thankyou 🙇

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -724,4 +724,49 @@ document.addEventListener('DOMContentLoaded', () => {
...
+  inputEmail.addEventListener('input', (event) => {
+    savedData.inputEmail = event.target.value;
+    localStorage.setItem('formData', JSON.stringify(savedData));
+  });
+
+  inputTextArea.addEventListener('input', (event) => {
+    savedData.inputTextArea = event.target.value;
+    localStorage.setItem('formData', JSON.stringify(savedData));
+  });
+
```
Comment (ABDULALI3468) [inline app.js:765]: Currently, this text area field is not updating the local storage when data is being added to it. Please consider resolving it to meet the project requirements. 


```
{
    "inputName": "ABDUL ALI",
    "inputEmail": "example@gmail.com"
}
```

**[OPTIONAL]** The project submits without even adding anything in the text area field consider making it required to make it more professional.

**Review by ABDULALI3468 — CHANGES_REQUESTED** [review]
# Changes Required 👍 

Hi @gisachris , 



Indeed you did an exceptional but still, there is room for changes to complete this project. 💯


## Highlights 
☑️Persisting data in local storage.
✅All linters are checked.
✅Project is professionally documented.
✅The right GitHub flow method has been used.
✅Professional pull request title and description.


 ## Required Changes ♻️

> **_Check the comments below to have a look at the required changes._**

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

## Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification.

> * This is My **GitHub**: GitHub followers
> * This is My **Linkedin**: [Linkedin Badge](https://www.linkedin.com/in/abdul-ali-5400bb216/)&nbsp;
> * This is My **Twitter**: [Twitter Badge](https://twitter.com/mrabdul_ali)&nbsp;

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app.js (hunk 1 above)
Comment (gisachris) [reply app.js:765]: ### **As per my previous CR 🔍  i was requested ♻️  to make some changes like :**
**-Add the email 📫  input to the local storage 🥇.**

as per my last commit 🟢  to the pull request i have fixed  🧑‍🔧  all the issues above. 🏅  
I hope to hear from you my CR 🤺  soon!.

**Review by 1GORDON — APPROVED** [review]
# Status: APPROVED 🟢 

Hi @gisachris & @ANTOINE1128 ✋

Your project is complete!  There is nothing else to say other than... it's time to merge it: ship it:
Congratulations! :tada: :clap: :clap: :100:

## Highlights!
- Followed Gitflow ✔️
- Good commit messages ✔️
- Linters are passing ✔️
- Descriptive Pull request ✔️

## Optional suggestions

- [ ] You may also fix the broken MIT license link to make your readme file more professional.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend considering them as they can improve your code._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@1GORDON` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**